src/window.rs

```diff
@@ -171,13 +171,14 @@ impl WindowBuilder {
 
     pub fn build(self) -> Window {
         let event_loop = winit::event_loop::EventLoop::new();
-        let mut builder = winit::window::WindowBuilder::new();
+        let mut builder = winit::window::WindowBuilder::new()
+            .with_min_inner_size(winit::dpi::Size::Logical((64, 64).into()));
```

Comment:
where does 64x64 come from?

Reply:
Just a random number :) 

We can expose it to user, or set to 1x1.

Reply:
Just curious, what do other apps do? I thought the default builder wouldn't give us 0x0

Reply:
Unfortunately, it give us 0x0.

```rust
thread 'main' panicked at 'Error in Surface::configure: Both `Surface` width and height must be non-zero. Wait to recreate the `Surface` until the window has non-zero area.'
```

But all works fine with minimal 1x1 size.

Reply:
I find it strange that a default `winit` configuration would produce 0x0, but your change is good